ksql-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java

```diff
@@ -182,6 +184,14 @@
           + "to ksql.streams.num.standby.replicas >= 1";
   public static final boolean KSQL_QUERY_PULL_ENABLE_STANDBY_READS_DEFAULT = false;
 
+  public static final String KSQL_QUERY_PULL_MAX_ALLOWED_OFFSET_LAG_CONFIG =
+      "ksql.query.pull.max.allowed.offset.lag";
+  public static final Long KSQL_QUERY_PULL_MAX_ALLOWED_OFFSET_LAG_DEFAULT = 0L;
```

Comment:
Should this be `LONG.MAX_VALUE`?

Reply:
`0` means, we don't query standbys at all (they will always have non-zero lag right).. So someone enabling standby.reads will be forced to also set this, to issue those queries.. I can imagine a lot of tickets we get where users complain standby routing is not working etc.. 

might be better to err on the other side? and as users become concerned about stale values, they will reach out and we point at this config.. wdyt

Reply:
Sure, I didn't really know if the default should be super low, super high, or some actual target in the middle.  I'm fine to do Long.MAX_VALUE as a default.